src/components/Inputs/TextField.scss

```diff
@@ -0,0 +1,18 @@
+@import "/src/theme/Theme.scss";
+
+.base-input {
+    display: flex;
+    outline: none;
+    width: 256px;
+    padding: 9px 17px;
+    align-items: flex-start;
+    background-color: $primary;
+    color: $on-primary;
+    gap: 10px;
+    border-radius: 20px;
+    border: 1px solid #FEFDFF80;
```

Comment:
ne uporabljaj svojih barv, uporabi barvo iz teme katera se ti zdi smiselna. Recimo $on-primary ali $primary-container

Reply:
To sem gli bil v dvomih, ko sn gledo po designi na Figmi. Tam je bla ena barva, ki je ni v temi oz., če je sem verjetno spregledal. Jo lahko dodam al bi mogle vse bit notri?

Reply:
design barv po figmi zaenkrat ne gledaj. Probaj čimveč uporabljat iz teme koliko se da, size, paddinge, spacinge, barve.

Boma pa malo spremenila design barv po svoje z koncom tedna

Reply:
Aha, okej. Nisem vedo pa sn se čimbolj designa probal držati